### DISCUSSION TITLE
Docker Hub OCI Repository with Autosync

### DISCUSSION THREAD

**Discussion by chayandashora29** [argoproj/argo-cd#19009] [Q&A]
I am using Docker Hub as custom OCI Registry with ArgoCD.
I have multiple versions of my application's helm chart uploaded to the registry.
I have enabled Auto-Sync in my application. However, when I push a new version into the docker hub, then argocd does not pick up that new version.
Is there any configuration that needs to be done for the same ?

**Comment by ChristianCiach** [comment]
You are probably hitting this bug:

* https://github.com/argoproj/argo-cd/issues/4002

There doesn't seem to be a solution at the moment.
